src/units.js

```diff
@@ -95,5 +94,15 @@ export const calculateSize = function(
             scale *= unitOptions.sizeMultiplier / options.sizeMultiplier;
         }
     }
-    return utils.round(Math.min(sizeValue.number * scale, options.maxSize));
+    return Math.min(sizeValue.number * scale, options.maxSize);
+};
+
+/**
+ * Round `n` to 4 decimal places, or to the nearest 1/10,000th em. The TeXbook
+ * gives an acceptable rounding error of 100sp (which would be the nearest
+ * 1/6551.6em with our ptPerEm = 10):
+ * http://www.ctex.org/documents/shredder/src/texbook.pdf#page=69
```

Comment:
This argument makes me a little uncomfortable, because that same passage says that TeX uses `1sp` as its unit, which is `1/65536 pt` or `1/655360 em` with our `ptPerEm = 10`, which would suggest we need 6 digits (instead of 4) to "match" TeX. The literal text is "the wavelength of visible light is approximately 100 sp, rounding errors of a few sp make no difference to the eye."  I think 100 sp *could* make a difference to the eye &mdash; that's like moving 1 "eye pixel". 🙂 

Then I wondered about device pixels. `1em` is `1/16 px`, and there are `devicePixelRatio` actual pixels per `px`.  I've typically seen `devicePixelRatio` ~ 2 for 4K displays.  `0.0001 = 1/(16*625)`, so we'd need a `devicePixelRatio` of `625` before 4-digit rounding moved by an entire pixel.  Said another way, with `devicePixelRatio = 2`, we shift things by ~0.3% of a pixel, which probably has minimal impact on antialiasing.

Makes sense: probably print is more sensitive than screens. Let's think about printing a KaTeX document with `1pt` mapped to `1pt` of the page. The highest resolution printer I could find is 5760 dpi, which is around 80 dots per pt, or 800 dots per em (at `ptPerEm = 10`).  This would require 3 digits of precision to get within a dot (roughly).  (I assume anti-aliasing doesn't happen at this resolution?)

Said another way, 4 digits of em precision = 3 digits of pt precision translates to 1000 dots per pt = 72270 dpi.  This does seem like an acceptable loss; even if we zoomed 10x, it's better than all existing printers.

Reply:
I've replaced it with the link to this PR for future reference.